documents/postman/internal_api_developers/notification-api.postman_collection.json

```diff
@@ -3272,7 +3272,7 @@
 								],
 								"body": {
 									"mode": "raw",
-									"raw": "{\r\n    \"sms_sender\": \"+18334981539\",\r\n    \"is_default\": false,\r\n    \"rate_limit\": 3000\r\n}"
+									"raw": "{\r\n    \"description\": \"used for xyz\",\r\n    \"provider_id\": \"uuid_of_provider\",\r\n    \"sms_sender\": \"+18334981539\",\r\n    \"is_default\": false,\r\n    \"rate_limit\": 3000\r\n}"
```

Comment:
` +18334981539` this is a real number. please use something fake, like `+12025551212`